javascript/javascript1/week4/voice-assistant/voice-assistant.js

```diff
@@ -0,0 +1,93 @@
+const dayjs = require('dayjs');
+const math = require('mathjs');
+
+function getReply(command) {
+  const storeName = "Benjamin";
+  const toDo = ["fishing", "singing in the shower"];
+  const myDate = dayjs();
+  const formatOfDate = myDate.format("dddd DD of MMMM YYYY, hh:mm A");
+  
+  switch (command) {
+    case `Hello my name is ${storeName}`:
+      return `Nice to meet you ${storeName}`;
+
+    case "What is my name?":
+      if (storeName.toLowerCase) {
+        return `Your name is ${storeName}`;
+      } else {
+        return "Your name has not been set yet";
+      }
+
+    case "What day is it today?":
+      return `Today is ${formatOfDate}`;
+
+    default:
+      if (command.startsWith("Add ")) {
+        const task = command.replace("Add ", "").replace(" to my todo", "");
+        toDo.push(task);
+        return `${task} added to your todo`;
+      }
+
+      if (command.startsWith("Remove ")) {
```

Comment:
You can move all the `command` conditionals up to the `case` statements from `default`.

`default` is often the final logic returned if the other conditionals aren't returned.